### PR TITLE
Update email confirmation

### LINKED ISSUE #303: Verify account with email after register
User can only log in into the plataform after the account is verified through email.

### PR DESCRIPTION
**Pull request by ruioliveira02**
Now the pages are styled and there is a plug which restricts access to routes to users that have already been confirmed via email

### CODE REVIEW

File: lib/atomic_web/router.ex [hunk 1]
```diff
@@ -155,9 +158,6 @@ defmodule AtomicWeb.Router do
 
   scope "/", AtomicWeb do
     pipe_through [:browser, :require_authenticated_user]
-
-    get "/users/settings", UserSettingsController, :edit
-    put "/users/settings", UserSettingsController, :update
     get "/users/settings/confirm_email/:token", UserSettingsController, :confirm_email
```
Comment (MarioRodrigues10) [inline lib/atomic_web/router.ex:161]: Is this route enough to confirm an account?
Shouldn't there be something like:
```
    get "/confirm", UserConfirmationController, :new
    post "/confirm", UserConfirmationController, :create
    get "/confirm/:token", UserConfirmationController, :edit
    post "/confirm/:token", UserConfirmationController, :update
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Also please make sure CI passes

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
After i register this logs me in instead of redirecting to login page.
And also when im logged in i can only acess general routes.

The flow should be:
Register -> Confirm email -> Login

File: lib/atomic_web/router.ex (hunk 1 above)
Comment (ruioliveira02) [reply lib/atomic_web/router.ex:161]: No

**Review by MarioRodrigues10 — APPROVED** [review]
:rocket:

**Pull request merged**